### PR TITLE
php2branch >Tl1revbranch

### PR DESCRIPTION
**Pull request by php-2**

### CODE REVIEW

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?php 
+
+class Firebase {
+ 
+    public function send($registration_ids, $message) {
+        $fields = array(
+            'registration_ids' => $registration_ids,
+            'data' => $message,
+        );
+        var_dump($fields);
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php]: Should it be only in debug mode?

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetAllUserToken.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetAllUserToken.php]: change this name apiAppFirebaseGetAllUserToken.php
to be api_fb_token.php

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebasePush.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebasePush.php]: change this name to apiAppFirebasePush.php
api_fb_push.php

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetTokenByEmail.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetTokenByEmail.php]: change file name to be api_fb_token_email.php

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php [hunk 5]
```diff
@@ -0,0 +1,62 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php]: api_fb.php

also please note in our folders and files we must have no capital letters.

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendMultiplePushByEmail.php [hunk 6]
```diff
@@ -0,0 +1,61 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendMultiplePushByEmail.php]: apiSendMultiplePushByEmail.php
api_sendmulti_push_email.php

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendSinglePushByEmail.php [hunk 7]
```diff
@@ -0,0 +1,54 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendSinglePushByEmail.php]: api_sendsingle_push_email.php

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/clsFirebaseNotification.php [hunk 8]
```diff
@@ -0,0 +1,145 @@
+<?php
```
Comment (hythamsoliman81) [inline portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/clsFirebaseNotification.php]: clsFirebaseNotification.php
cls_fb_notification.php

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php (hunk 1 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php]: i did that

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendMultiplePushByEmail.php (hunk 6 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendMultiplePushByEmail.php]: done

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendSinglePushByEmail.php (hunk 7 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiSendSinglePushByEmail.php]: done

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/clsFirebaseNotification.php (hunk 8 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/clsFirebaseNotification.php]: done

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php (hunk 5 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebase.php]: i changed it

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetTokenByEmail.php (hunk 4 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetTokenByEmail.php]: i renamed it

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebasePush.php (hunk 3 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebasePush.php]: done

File: portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetAllUserToken.php (hunk 2 above)
Comment (php-2) [reply portfolio/ifintech/v1/ierp/htdocs/custom/extintegration/class/firebase/apiAppFirebaseGetAllUserToken.php]: done

**Pull request merged**